terraform/cloudformation/asg.json.template

```diff
@@ -1,45 +0,0 @@
-{
-  "Resources": {
```

Comment:
Are we just not using this any more?

Reply:
Yup. It got moved inside another module and we didn't delete this one - should probably be another PR, but low hanging fruit.